support/src/main/java/se/inera/intyg/common/support/model/InternalDate.java

```diff
@@ -202,6 +202,11 @@ public boolean isBeforeBeginningOfLastYear() {
             && this.asLocalDate().isBefore(LocalDate.ofYearDay(LocalDate.now().getYear() - 1, 1));
     }
 
+    public boolean isBeforePatientsBirthDate(LocalDate patientBirthDate) {
```

Comment:
Since we're sending the birthday this function is only comparing if the date is before the date sent as a parameter, so we can make it more general

Reply:
Great suggestion, changed it to isBefore().